src/main/java/com/google/devtools/build/lib/bazel/rules/python/stub_template.txt

```diff
@@ -12,6 +12,20 @@ import zipfile
 def IsWindows():
   return os.name == 'nt'
 
+def FixAbsWindowsPath(path):
```

Comment:
That means that the path in question must NEVER be extended afterwards. Also, nobody guarantees that some Python API does not choke on 258 characters or such. In general it makes the whole handling way more brittle IMO.